data-reconciliation-app/test/e2e-test/src/api.ts

```diff
@@ -0,0 +1,41 @@
+import { DemoMemberProps, DemoProps } from './index';
+import axios from 'axios';
+import { SummaryRecordProps } from '../../../src/models/summary-record';
+
+export interface ReportItem {
+    group_status: string;
+    majority_minority: string;
+    count_of_unique_values: number;
+    members_in_agreement: number;
+    lei: string;
+    nace: string;
+}
+
+export default class Api {
+    public static async ingest(props: DemoProps, member: DemoMemberProps) {
+        console.log(`📝 ${member.name} Ingesting Data...`);
+
+        const result = await axios.post(props.ingestUrl, member.data, { httpsAgent: member.httpsAgent });
+
+        if(result.status !== 200) {
+            throw new Error(`🛑 [TEST FAILURE]: Unexpected status code: ${result.status}`);
```

Comment:
Throwing an error will fail the `make` command and I think this will be reflected to CI/CD actions too.